example-config.yaml

```diff
@@ -54,6 +56,8 @@ chat:
   users_mapping:
     # Mapping between Project owners login and Slack users.
     akieras: akieras
+  # Minimum Cost to notify the owner
+  cost_min_to_notify: 0.0
```

Comment:
Move this to line 52.
Change comment to "Cost minimum to actually send a message (notify) to the owner."

Reply:
these changes are made in commit 19ae898